common/models/user.js

```diff
@@ -752,10 +762,10 @@ module.exports = function(User) {
       {
         description: 'Logout a user with access token.',
         accepts: [
-          {arg: 'access_token', type: 'string', required: true, http: function(ctx) {
```

Comment:
Why is 400 worse than 401 if the token is not provided? IMO, it's better to report it as missing argument.